flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/BroadcastStateTransformation.java

```diff
@@ -53,111 +50,91 @@
 @Internal
 public class BroadcastStateTransformation<IN1, IN2, OUT> extends PhysicalTransformation<OUT> {
```

Comment:
The two BroadcastTransformations could have a base class without the key-related fields and the function.